packages/extension-commerce-bm-seo/README.md

```diff
@@ -0,0 +1,124 @@
+                {
+                    "enabled": true,
+                    "commerceAPI": {
+                    "proxyPath": "/mobify/proxy/api",
+                    "parameters": {
+                        "shortCode": "8o7m175y",
+                        "clientId": "c9c45bfd-0ed3-4aa2-9971-40f88962b836",
+                        "organizationId": "f_ecom_zzrf_001",
+                        "siteId": "RefArchGlobal"
+                    }
+                    },
+                    "commerceAPIAuth": {
+                    "propertyNameInLocals": "commerceAPIAuth"
+                    },
+                    "resourceTypeToComponentMap": {
+                    "category": "ProductList",
+                    "product": "ProductDetail",
+                    }
+                }
+                ]
+            ]
+        }
+    }
+}
+```
+
+Please note that the this extension needs to be defined in the array before any other extension that uses `isNavigationBlocked` to
+toggle the rendering of `<WrappedComponent />` in the extension's HOC. Otherwise, `extension-commerce-bm-seo` will not be able to
+set `isNavigationBlocked` back to a state to allow the rendering of `<WrappedComponent />` since its logic is within the component. 
+
+### Configuration Options
+
+- `commerceAPI`: Settings for connecting to the Commerce API
+   - `proxyPath`: The proxy path for API requests
+   - `parameters`: Commerce API connection parameters
+     - `clientId`: Your Commerce API client ID
+     - `organizationId`: Your organization ID
+     - `shortCode`: Your short code
+     - `siteId`: Your site ID
+ - `commerceAPIAuth`: Authentication configuration
+   - `propertyNameInLocals`: Property name for storing auth in locals
+ - `resourceTypeToComponentMap`: Maps resource types to component names
+   - `category`: Component name for category pages
+   - `product`: Component name for product pages
+   - `content_asset`: Component name for content assets
+
+## How It Works
+
+The SEO extension works by:
+1. Intercepting incoming requests
+2. Querying the Shopper SEO API for URL mappings
+3. Dynamically routing to the appropriate component based on the resource type
+4. Passing the relevant props to the component
+
+The extension uses the Commerce API's Shopper SEO URL Mapping endpoint to handle SEO-friendly URLs and ensure proper routing within your PWA Kit application.
+
+## Usage of `isNavigationBlocked` and `siteLocale`
+
+### isNavigationBlocked
+
+- The SEO extension provides an `isNavigationBlocked` state via the application extension store.
+- Downstream extensions (such as `@salesforce/extension-chakra-storefront`) should read this value to determine whether navigation is being blocked (e.g., while waiting for SEO URL mapping to resolve).
+- When `isNavigationBlocked` is `true`, the downstream extension can use the state to display a loading state until the SEO extension has finished processing the URL mapping.
```

Comment:
There is no requirement that extensions are "downstream". So we might want to use something like "sibling" or simply "other installed extensions"